src/test/config.sol

```diff
@@ -78,9 +87,50 @@ contract Config {
     uint256 constant MILLION    = 10 ** 6;
     uint256 constant BILLION    = 10 ** 9;
 
+    uint256 constant monthly_expiration = 4 days;
+    uint256 constant weekly_expiration  = 30 days;
+
+    SpellValues  spellValues;
     SystemValues afterSpell;
 
-    function setCollateralValues() public {
+    function setValues(address chief) public {
```

Comment:
Why is the chief address a param ? It will be the same every time this function is called.

Is this done in order to avoid importing `addresses_mainnet.sol` and `DssInterfaces` to this file ?

Reply:
Yeah it was mainly to do it in an easy way. For sure is not the most elegant way and we can do it better. So we could evaluate other alternatives.